puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/build_deb_coreproject.yaml

```diff
@@ -1,6 +1,6 @@
 - builder:
     name: build_deb_coreproject
     builders:
-      - shell: !include-raw scripts/debian/configure_pbuilder.sh
-      - shell: !include-raw scripts/debian/execute_pbuilder.sh
-      - shell: !include-raw scripts/debian/stage_packages.sh
+      - shell: !include-raw: scripts/debian/configure_pbuilder.sh
```

Comment:
Nope, ignore that - I hadn't run easy_install, looks fine. I'll re-run Travis CI.